hazelcast-hibernate53/src/test/resources/hazelcast-custom.xml

```diff
@@ -30,36 +30,10 @@
                 <interface>127.0.0.1</interface>
             </tcp-ip>
         </join>
-        <interfaces enabled="false">
-            <interface>10.3.17.*</interface>
-        </interfaces>
     </network>
-    <map name="default">
-        <!--
-            Number of backups. If 1 is set as the backup-count for example,
-            then all entries of the map will be copied to another JVM for
-            fail-safety. Valid numbers are 0 (no backup), 1, 2, 3.
-        -->
-        <backup-count>1</backup-count>
-        <!--
-        eviction-policy:
-            Valid values are:
-            NONE (no eviction),
-            LRU (Least Recently Used),
-            LFU (Least Frequently Used).
-            NONE is the default.
-        size:
-            Maximum size of the map. When max size is reached,
-            map is evicted based on the policy defined.
-            Any integer between 0 and Integer.MAX_VALUE. 0 means
-            Integer.MAX_VALUE. Default is 0.
-        -->
-        <eviction eviction-policy="NONE" size="0" />
-    </map>
 
-    <!-- Config for query cache -->
     <map name="default-query-results-region">
-        <time-to-live-seconds>0</time-to-live-seconds>
+        <time-to-live-seconds>2</time-to-live-seconds>
```

Comment:
Is that still needed?

Reply:
Yes, because I want them all to expire quite fast

Reply:
I couldn't spot any test of which duration is based on this value tho. Total time elapsed is almost the same with or without this value. Is there a point I'm not aware of?

Reply:
Um, you're right :) I was thinking about the different config file. Will roll it back